packages/cli/src/commands/generate/sdl/__tests__/fixtures/enumGeneratedSdl.js

```diff
@@ -0,0 +1,33 @@
+export const schema = gql`
+  type Shoe {
+    id: Int!
+    color: Color!
+  }
+
+  enum Color {
+    RED
+
+    GREEN
+
+    BLUE
+  }
+
+  type Query {
+    shoes: [Shoe!]!
+    shoe(id: Int!): Shoe!
+  }
+
+  input CreateShoeInput {
+    color: Color!
+  }
+
+  input UpdateShoeInput {
+    color: Color
+  }
+
+  type Mutation {
+    createShoe(input: CreateShoeInput!): Shoe!
+    updateShoe(id: Int!, input: UpdateShoeInput!): Shoe!
+    deleteShoe(id: Int!): Shoe!
+  }
+`;
```

Comment:
Redwood's standard coding style does not use semicolons...are you using our prettier/linter settings? They should automatically format everything to our style on save.